src/view/actions/resetEventMergeId.jsx

```diff
@@ -16,9 +16,9 @@ import { object, string } from "yup";
 import Textfield from "@react/react-spectrum/Textfield";
 import FieldLabel from "@react/react-spectrum/FieldLabel";
 import "@react/react-spectrum/Form"; // needed for spectrum form styles
-import render from "../render";
+import render from "../spectrum2Render";
```

Comment:
Some of these files, like `render.jsx`, are now different between the React-Spectrum-2/Formik views and the React-Spectrum-3/React-Hook-Form views. To keep a clear distinction, I named some of the files accordingly. Once we're fully off React-Spectrum-2/Formik, we'll probably rename them back to something more generic like `render.jsx`.